src/pages/Book/index.js

```diff
@@ -51,3 +50,5 @@ export class Book extends React.Component {
     );
   }
 }
+
+export default Book;
```

Comment:
here as well, you can add "export default" right before you declare the class component